apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -210,21 +210,6 @@ defmodule Aecore.Miner.Worker do
     top_block_hash = BlockValidation.block_header_hash(top_block.header)
     chain_state = Chain.chain_state(top_block_hash)
 
-    # We take an extra block and then drop one at the head of the list
```

Comment:
why is this not needed any more?

Reply:
We take the blocks a few lines down, I don't know why that was still there